assets/xml/objects/object_dnt.xml

```diff
@@ -1,25 +1,25 @@
 ﻿<Root>
     <File Name="object_dnt" Segment="6">
-        <Animation Name="object_dnt_Anim_000994" Offset="0x994" />
-        <Animation Name="object_dnt_Anim_0012F4" Offset="0x12F4" />
+        <Animation Name="object_dnt_Anim_000994" Offset="0x994" /> <!-- Original name is "dnt_banban" -->
+        <Animation Name="object_dnt_Anim_0012F4" Offset="0x12F4" /> <!-- Original name is "dnt_boyon" -->
         <DList Name="object_dnt_DL_001350" Offset="0x1350" />
         <DList Name="object_dnt_DL_001420" Offset="0x1420" />
-        <Animation Name="object_dnt_Anim_001BC8" Offset="0x1BC8" />
-        <Animation Name="object_dnt_Anim_001E2C" Offset="0x1E2C" />
-        <Animation Name="object_dnt_Anim_002268" Offset="0x2268" />
-        <Animation Name="object_dnt_Anim_002670" Offset="0x2670" />
-        <Animation Name="object_dnt_Anim_0029E8" Offset="0x29E8" />
-        <Animation Name="object_dnt_Anim_002F08" Offset="0x2F08" />
-        <Animation Name="object_dnt_Anim_003438" Offset="0x3438" />
-        <Animation Name="object_dnt_Anim_0038CC" Offset="0x38CC" />
-        <Animation Name="object_dnt_Anim_003CC0" Offset="0x3CC0" />
+        <Animation Name="object_dnt_Anim_001BC8" Offset="0x1BC8" /> <!-- Original name is "dnt_fly" -->
+        <Animation Name="object_dnt_Anim_001E2C" Offset="0x1E2C" /> <!-- Original name is "dnt_iyaiya" -->
+        <Animation Name="object_dnt_Anim_002268" Offset="0x2268" /> <!-- Original name is "dnt_iyaiyaTOmuun" -->
+        <Animation Name="object_dnt_Anim_002670" Offset="0x2670" /> <!-- Original name is "dnt_kouka" -->
+        <Animation Name="object_dnt_Anim_0029E8" Offset="0x29E8" /> <!-- Original name is "dnt_moguru" -->
+        <Animation Name="object_dnt_Anim_002F08" Offset="0x2F08" /> <!-- Original name is "dnt_muun" -->
+        <Animation Name="object_dnt_Anim_003438" Offset="0x3438" /> <!-- Original name is "dnt_nooo" -->
+        <Animation Name="object_dnt_Anim_0038CC" Offset="0x38CC" /> <!-- Original name is "dnt_onedari" -->
+        <Animation Name="object_dnt_Anim_003CC0" Offset="0x3CC0" /> <!-- Original name is "dnt_onedariTOwait" -->
         <Blob Name="object_dnt_Blob_003CD0" Size="0x10" Offset="0x3CD0" />
-        <Animation Name="object_dnt_Anim_004700" Offset="0x4700" />
-        <Animation Name="object_dnt_Anim_004AA0" Offset="0x4AA0" />
-        <Animation Name="object_dnt_Anim_004E38" Offset="0x4E38" />
-        <Animation Name="object_dnt_Anim_005488" Offset="0x5488" />
-        <Animation Name="object_dnt_Anim_00577C" Offset="0x577C" />
-        <Animation Name="object_dnt_Anim_005CA8" Offset="0x5CA8" />
+        <Animation Name="object_dnt_Anim_004700" Offset="0x4700" /> <!-- Original name is "dnt_spoon" -->
```

Comment:
wat

Reply:
Yeah it looks like that's the correct animation. I have absolutely zero idea why it is called "spoon"
![spoon](https://user-images.githubusercontent.com/12245827/207517207-b0b7edaa-2223-434c-90ee-5e299fe18753.gif)